templates/frontend/objects/article_details.tpl

```diff
@@ -212,24 +212,26 @@
 							{translate key="submission.authorBiography"}
 						{/if}
 					</h2>
+					<ul class="authors">
 					{foreach from=$publication->getData('authors') item=author}
 						{if $author->getLocalizedData('biography')}
-							<section class="sub_item">
-								<h3 class="label">
+							<li class="sub_item">
+								<span class="label">
 									{if $author->getLocalizedData('affiliation')}
 										{capture assign="authorName"}{$author->getFullName()|escape}{/capture}
 										{capture assign="authorAffiliation"}<span class="affiliation">{$author->getLocalizedData('affiliation')|escape}</span>{/capture}
 										{translate key="submission.authorWithAffiliation" name=$authorName affiliation=$authorAffiliation}
 									{else}
 										{$author->getFullName()|escape}
 									{/if}
-								</h3>
-								<div class="value">
+								</span>
+								<span class="value">
 									{$author->getLocalizedData('biography')|strip_unsafe_html}
-								</div>
-							</section>
+								</span>
```

Comment:
Why change this to `<span>`?

Reply:
Good catch!! I probably brought the element from the list on the top. However, we need to keep it using `DIV` because it is a fallback for bios without the `P` tag and we force a block element to separate from the previous element. I will adjust it.